### ISSUE TITLE
Android Gradle failing to compile

### ISSUE THREAD

**Issue by zwells** [vanlooverenkoen/flutter_appcenter_release_manager#38]
I think gradle version needs updated again?

```FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':appcenter_release_manager'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Please specify a namespace in the module's build.gradle file like so:

     android {
         namespace 'com.example.namespace'
     }

     If the package attribute is specified in the source AndroidManifest.xml, it can be migrated automatically to the namespace value in the build.gradle file using the AGP Upgrade Assistant; please refer to https://developer.android.com/studio/build/agp-upgrade-assistant for more information.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 18s

**Comment by vanlooverenkoen** [comment]
Thanks for bringing this to my attention. You can create a PR if you want. Otherwise I will add it to my todo list

**Comment by zwells** [comment]
Downgraded to gradle 7.4.2 for now

**Comment by vanlooverenkoen** [comment]
Should be fixed! @zwells can you test with 4.0.0?
